proto/internal/temporal/server/api/adminservice/v1/service.proto

```diff
@@ -65,8 +65,12 @@ service AdminService {
     rpc ReapplyEvents (ReapplyEventsRequest) returns (ReapplyEventsResponse) {
     }
 
-    // AddSearchAttribute whitelist search attribute in request.
-    rpc AddSearchAttribute (AddSearchAttributeRequest) returns (AddSearchAttributeResponse) {
+    // AddSearchAttribute add custom search attributes and returns comprehensive information about them.
+    rpc AddSearchAttributes (AddSearchAttributesRequest) returns (AddSearchAttributesResponse) {
```

Comment:
AddSearchAttribute -> AddSearchAttributes

compatibility issue?

Reply:
Yes, It is a breaking change. I need to add this to PR description. But given that:
1. This API was always broken,
2. I had to change request and response objects too, 
I believe renaming it, is a good idea.